R/ui.R

```diff
@@ -153,6 +149,15 @@ HarmonyMatrix <- function(
     }
     
     # determine if lambda is appropriate
+    if (!("lambda" %in% names(opt_args))){
```

Comment:
a better function to test this condition is `hasArgs()`

Reply:
Yes you are right, I will change it